client/src/pages/Search/MapView/RestaurantPreview.js

```diff
@@ -1,7 +1,103 @@
-function RestaurantPreview({ restaurant }) {
+import React from "react";
+import ReviewStars from "../../../components/ReviewStars";
+import { IconButton } from "@mui/material";
+import CancelIcon from "@mui/icons-material/Cancel";
+import { Link } from "react-router-dom";
+
+const classes = {
+  previewContainer: {
+    height: "120px",
+    width: "100%",
+    backgroundColor: "rgb(224,224,224)",
+    display: "flex",
+    justifyContent: "space-between",
+  },
+  imageContainer: {
+    height: "75%",
```

Comment:
try height 100%

Reply:
does not work, lol. looks jank with 100%.